app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <ImageView
+        android:id="@+id/vHeaderImage"
+        android:layout_width="match_parent"
+        android:layout_height="204dp"
+        android:layout_marginBottom="68dp"
+        android:scaleType="fitXY"
+        android:background="@drawable/login_cover"
+        android:src="@drawable/wolox_colors_gradient"
+        app:layout_constraintTop_toTopOf="parent"></ImageView>
+
+    <RelativeLayout
```

Comment:
you don't need to use a relative layout to fit a image. You can just use the imageView. In case you think this is the better way, we can talk about it. I suggest you to avoid using relative layout